app/src/main/java/com/framgia/toeic/screen/splash/FragmentAdapter.java

```diff
@@ -0,0 +1,104 @@
+import android.view.WindowManager;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.framgia.toeic.R;
+
+public class FragmentAdapter extends PagerAdapter {
+    private Context mContext;
+    private onNext mOnNext;
+    private ConstraintLayout mLayoutIntro;
+    private ImageView mImageIntro;
+    private TextView mTitleIntro;
+    private TextView mDesIntro;
+    private Button mButtonIntro;
+
+    public FragmentAdapter(Context context) {
+        this.mContext = context;
+    }
+
+    private static final int[] images = {
+            R.drawable.bg_intro_5,
+            R.drawable.bg_intro_6,
+            R.drawable.bg_intro_7
+    };
+
+    private static final int[] titles = {
+            R.string.title_intro_1,
+            R.string.title_intro_2,
+            R.string.title_intro_3
+    };
+
+    private static final int[] descriptions = {
+            R.string.des_intro_1,
+            R.string.des_intro_2,
+            R.string.des_intro_3
+    };
+
+    private static final int[] backgroundcolors = {
+            R.color.material_red_400,
+            R.color.material_brown_600,
+            R.color.material_cyan_500
+    };
+
+    private static final int[] lst_button = {
+            R.string.button_intro_1,
+            R.string.button_intro_1,
+            R.string.button_intro_2
+    };
+
+    @Override
+    public int getCount() {
+        return titles.length;
+    }
+
+    @Override
+    public boolean isViewFromObject(View view, Object object) {
+        return (view == object);
+    }
+
+    @Override
+    public Object instantiateItem(ViewGroup container, final int position) {
+        View view = LayoutInflater.from(mContext).inflate(R.layout.fragment_intro, container, false);
+        mLayoutIntro = view.findViewById(R.id.layout);
+        mImageIntro = view.findViewById(R.id.image_intro);
+        mTitleIntro = view.findViewById(R.id.text_title_intro);
+        mDesIntro = view.findViewById(R.id.text_des_intro);
+        mButtonIntro = view.findViewById(R.id.button_intro);
+        addItem(position);
+        container.addView(view);
+        return view;
+    }
+    public void addItem(final int position){
+        mOnNext = (onNext) mContext;
+        mButtonIntro.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                mOnNext.position(mButtonIntro.getText().toString(), position);
+            }
+        });
+        mLayoutIntro.setBackgroundResource(backgroundcolors[position]);
+        mImageIntro.setImageResource(images[position]);
+        mTitleIntro.setText(titles[position]);
+        mDesIntro.setText(descriptions[position]);
+        mButtonIntro.setText(lst_button[position]);
+    }
+
+    @Override
+    public void destroyItem(ViewGroup container, int position, Object object) {
+        container.removeView((View) object);
+    }
+
+    public interface onNext {
```

Comment:
tên class interface nên đặt là 1 danh từ
```suggestion
    public interface OnPageChangeListener {
```